src/DotNetWorker.Core/Context/Features/DefaultFunctionInputBindingFeature.cs

```diff
@@ -69,18 +70,29 @@ public async ValueTask<FunctionInputBindingResult> BindFunctionInputAsync(Functi
                     }
                     else
                     {
-                        IReadOnlyDictionary<string, object> properties = ImmutableDictionary<string, object>.Empty;
+                        var properties = new Dictionary<string, object>();
 
                         // Pass info about specific input converter type defined for this parameter, if present.
                         if (param.Properties.TryGetValue(PropertyBagKeys.ConverterType, out var converterTypeAssemblyFullName))
                         {
-                            properties = new Dictionary<string, object>()
+                            properties.Add(PropertyBagKeys.ConverterType, converterTypeAssemblyFullName);
+                        }
+
+                        // Pass info about the flag to allow fallback to default converters defined for this parameter, if present.
+                        if (param.Properties.TryGetValue(PropertyBagKeys.AllowConverterFallback, out var flag))
+                        {
+                            properties.Add(PropertyBagKeys.AllowConverterFallback, flag);
+                        }
+
+                        // Pass info about input converter types defined for this parameter, if present.
+                        if (param.Properties.TryGetValue(PropertyBagKeys.BindingAttributeSupportedConverters, out var converters))
                         {
-                            { PropertyBagKeys.ConverterType, converterTypeAssemblyFullName }
-                        };
+                            properties.Add(PropertyBagKeys.BindingAttributeSupportedConverters, converters);
                         }
 
-                        var converterContext = _converterContextFactory.Create(param.Type, source, context, properties);
+                        var converterContext = _converterContextFactory.Create(param.Type, source, context, properties.Count() != 0 
+                                                                                                ? properties.ToImmutableDictionary()
+                                                                                                : ImmutableDictionary<string, object>.Empty);
```

Comment:
nit: love having the ? and : aligned with the =  (i think it makes it more readable :) )

```
var converterContext = _converterContextFactory.Create(param.Type, source, context, properties.Count() != 0 
                     ? properties.ToImmutableDictionary()
                     : ImmutableDictionary<string, object>.Empty);
```